controllers/etcd_controller.go

```diff
@@ -991,6 +1148,7 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 		"statefulsetReplicas":     statefulsetReplicas,
 		"serviceName":             fmt.Sprintf("%s-client", etcd.Name),
 		"configMapName":           fmt.Sprintf("etcd-bootstrap-%s", string(etcd.UID[:6])),
+		"cronJobName":             fmt.Sprintf("%s-compact-cronjob", etcd.Name),
```

Comment:
```suggestion
		"cronJobName":             fmt.Sprintf("%s-compact-backup", etcd.Name),
```
This logic is duplicated. Can have it one place?

Reply:
Added a function `getCronJobName` that given the etcd object returns the cronjob name corresponding to that object